### PR TITLE
Add paper 1.19.4 support

### PR DESCRIPTION
**Pull request by TheMeinerLP**
## Overview
Yersterday paper released the wip state of 1.19.4. I pre-compiled already and ported the code to 1.19.4 
A pre-compiled 1.19.4 paper version comes with fawe this following error:
https://paste.gg/p/anonymous/27517a721e5c41069efc1dfa82e470f8
 

## Description
Paperweight 1.19.4 port and support

## Submitter Checklist
- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] New public fields and methods are annotated with `@since TODO`.
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by TheMeinerLP** [comment]
**Important** Its use a local m2 for compiling in very early stage of paper

**Comment by NotMyFault** [comment]
Thanks for the initial work. I went ahead and replaced your local paperweight version with the official release.

File: worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_19_R3/PaperweightAdapter.java [hunk 1]
```diff
@@ -0,0 +1,1016 @@
...
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+
+public final class PaperweightAdapter implements BukkitImplAdapter {
```
Comment (AnttiMK) [inline worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_19_R3/PaperweightAdapter.java:166]:
```suggestion
public final class PaperweightAdapter implements BukkitImplAdapter<net.minecraft.nbt.Tag> {
```
Missing type parameter

File: worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/regen/PaperweightRegen.java [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+import com.fastasyncworldedit.core.queue.IChunkCache;
+import com.fastasyncworldedit.core.queue.IChunkGet;
+import com.fastasyncworldedit.core.util.ReflectionUtils;
+import com.fastasyncworldedit.core.util.TaskManager;
+import com.google.common.collect.ImmutableList;
+import com.mojang.datafixers.util.Either;
+import com.mojang.serialization.Lifecycle;
+import com.sk89q.worldedit.bukkit.WorldEditPlugin;
+import com.sk89q.worldedit.bukkit.adapter.Refraction;
+import com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_19_R2.PaperweightGetBlocks;
```
Comment (AnttiMK) [inline worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/regen/PaperweightRegen.java:14]:
```suggestion
import com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_19_R3.PaperweightGetBlocks;
```
Invalid import

**Review by AnttiMK — CHANGES_REQUESTED** [review]
Tried to build the branch for testing, but stumbled upon some compilation errors that might need addressing.

**Comment by NotMyFault** [comment]
Hey,

thanks for your PR and your contributions @AnttiMK. I created a branch based on your work in this repository: https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/2154

@TheMeinerLP you forked FAWE to an organization instead of your personal account. Hence I'm unable to push anything to your organization.

**Comment by TheMeinerLP** [comment]
Thanks for the tip. How can i change that for future requests ?

**Comment by NotMyFault** [comment]
> Thanks for the tip. How can i change that for future requests ?

If you are on https://github.com/IntellectualSites/FastAsyncWorldEdit/fork, select a repository owner on the dropdown list, that is not an organization.

**Comment by TheMeinerLP** [comment]
And with organization is not working ?

**Comment by NotMyFault** [comment]
Not in a way maintainers can contribute back. Hence you don’t contribute from organisations directly, rather have it on a personal account.

**Pull request closed without merging**